### ISSUE TITLE
Snap and Debian checkbox produces inconsistent `fwts klog` result

### ISSUE THREAD

**Issue by tomli380576** [canonical/checkbox#1997]
### Bug Description

`fwts klog` reports 0 errors when running inside a checkbox shell but reports multiple errors when it's running as a regular debian package. The machine that can consistently reproduce this is 202411-35954.

[debian-fwts.log](https://github.com/user-attachments/files/21051934/results.log)

[snap-fwts.log](https://github.com/user-attachments/files/21051935/snap-fwts.log)

### Cert-blocker Test Case

- [ ] cert-blocker

### To Reproduce

1. Boot into OS
2. `sudo apt install fwts` and run `fwts klog`. This should produce an error
3. `sudo apt remove fwts` or just reinstall the OS
4. Install snap checkbox
5. `sudo -E checkbox.shell`
6. Run `fwts klog` again,this time there's no error

### Expected Result

fwts klog should report the same result in both environments

### Actual Result

fwts klog only produces errors in debian checkbox

### Environment

- Ubuntu24.04
- Both snap and debian

### Relevant log output

```shell

```
